src/gt4py/next/iterator/embedded.py

```diff
@@ -541,7 +548,7 @@ def promote_scalars(val: CompositeOfScalarOrField):
     }
     decorator = getattr(builtins, math_builtin_name).register(EMBEDDED)
     impl: Callable
-    if math_builtin_name in ["gamma", "not_"]:
```

Comment:
Can be put in the list above. The operator impl also works for columns out-of-the-box.